src/openproduct/producten/tests/test_metrics.py

```diff
@@ -0,0 +1,73 @@
+import datetime
+from unittest.mock import MagicMock, patch
+
+from django.test import override_settings
+from django.urls import reverse
+
+from openproduct.producten.metrics import (
+    product_create_counter,
+    product_delete_counter,
+    product_update_counter,
+)
+from openproduct.producten.tests.factories import ProductFactory, ProductTypeFactory
+from openproduct.urn.models import UrnMappingConfig
+from openproduct.utils.tests.cases import BaseApiTestCase
+
+
+@override_settings(NOTIFICATIONS_DISABLED=True, PRODUCTEN_API_MAJOR_VERSION=0)
+class ProductMetricsTests(BaseApiTestCase):
+    def setUp(self):
+        super().setUp()
+        self.producttype = ProductTypeFactory.create(
+            toegestane_statussen=["gereed"],
+            publicatie_start_datum=datetime.date(2024, 1, 1),
+        )
+
+        UrnMappingConfig.objects.create(
+            urn="maykin:abc:ztc:zaak",
+            url="https://maykin.ztc.com/api/v1/zaken",
+        )
+
+        self.data = {
+            "producttype_uuid": self.producttype.uuid,
+            "status": "initieel",
+            "prijs": "20.20",
+            "frequentie": "eenmalig",
+            "eigenaren": [{"kvk_nummer": "12345678"}],
+            "aanvraag_zaak_urn": "maykin:abc:ztc:zaak:d42613cd-ee22-4455-808c-c19c7b8442a1",
+        }
+
+    @patch.object(product_create_counter, "add", wraps=product_create_counter.add)
+    def test_product_create_counter(self, mock_add: MagicMock):
+        self.client.post(reverse("product-list"), self.data)
+
+        mock_add.assert_called_once_with(1)
+
+    @patch.object(product_update_counter, "add", wraps=product_update_counter.add)
+    def test_product_update_counter(self, mock_add: MagicMock):
```

Comment:
applies for delete too
```suggestion
    def test_product_update_counter(self, mock_update: MagicMock):
```

Reply:
nvm, didn't notice it's the `.add` that's being mocked everywhere